owa/app/js/components/manage_autogeneration_option/ManageAutoGenerationOption.jsx

```diff
@@ -0,0 +1,451 @@
+import React, { Component } from 'react';
+import { Button, Container, Row, Col, FormGroup, Label, Input, Table,
+    Modal, ModalHeader, ModalBody, ModalFooter, Alert } from 'reactstrap';
+import { MdDelete } from 'react-icons/lib/md';
+import './css/manage-autogeneration.css';
+
+class ManageAutoGenerationOption extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      genOptions: [
+        {
+          idType: 'OpenMRS ID',
+          location: 'Armani Hospital',
```

Comment:
Why are we hard coding these?

Reply:
Just to demo the feature. It will be replaced with data coming from the backend once the restful API's are created.